### PR TITLE
Mobile layout

### PR DESCRIPTION
**Pull request by neokore**
Related to https://app.clubhouse.io/cartoteam/story/121312/mobile-support-sample-app-and-skeleton

### CODE REVIEW

**Comment by VictorVelarde** [comment]
The space for the carto username is a bit short 
image, also it appears as a link but but no pointer.

Maybe we could just move the username into the dialog itself, more like in 
image, but simplified

**Comment by VictorVelarde** [comment]
Mobile looks really nice, great work!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # CHANGELOG
+## Unreleased
+- Adapt layout for mobile
```
Comment (VictorVelarde) [inline CHANGELOG.md:3]:
```suggestion
- Adapt layout for mobile [#134](https://github.com/CartoDB/carto-react-template/pull/134)
```

File: template-skeleton/template/src/components/views/Main.js [hunk 2]
```diff
@@ -1,69 +1,138 @@
...
-        direction='column'
-        alignItems='stretch'
-        item
-        className={classes.sidebarWrapper}
-      >
-        <Outlet />
-      </Grid>
+    <Grid container direction='row' alignItems='stretch' item xs>
+      <nav className={classes.drawer}>
+        <Portal container={isMobile ? mobileContainer.current : desktopContainer.current}>
```
Comment (VictorVelarde) [inline template-skeleton/template/src/components/views/Main.js:93]: Interesting use of Portal here... :D

**Comment by neokore** [comment]
> The space for the carto username is a bit short
> image, also it appears as a link, with no pointer, that I'm not sure about.
> 
> Maybe we could just move the username into the dialog itself, more like in
> image, but simplified

Yesterday I undertood that the design team want to create a better design, so by now let's keep this text from wrapping and then let's apply their new design when ready.

**Review by VictorVelarde — APPROVED** [review]
Feel free to merge after login username fix

**Pull request merged**